contrib/nuke-graph-directory.sh

```diff
@@ -61,10 +61,12 @@ if command -v btrfs > /dev/null 2>&1; then
 	# Source: http://stackoverflow.com/a/32865333
 	for subvol in $(find "$dir" -type d -inum 256 | sort -r); do
 		if [ "$dir" != "$subvol" ]; then
```

Comment:
Oh, it's probably worth bringing that conditinoal all the way up to here, ala:

```suggestion
		if [ "$dir" != "$subvol" ] && subvolType="$(stat -f --format=%T "$subvol")" && [ "$subvolType" = "btrfs" ]; then
```

(So we only have a single `if`)

Reply:
that's, nifty; and looks good to me.